eventhubs-streamanalytics-eventhubs/tools/eh-asa-perfmon/EventReceiver.cs

```diff
@@ -0,0 +1,119 @@
+﻿using Azure.Messaging.EventHubs.Consumer;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace StreamingAtScale
+{
+    public class EventReceiver
+    {
+        private readonly object _lock = new object();
```

Comment:
Please see below;  I think that we can safely remove this.

Reply:
Resolved